schemas/nl.basjes/yauaa_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a context generated by the YAUAA enrichment after parsing the user agent",
+    "self": {
+        "vendor": "nl.basjes",
+        "name": "yauaa_context",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "deviceClass": {
+            "description": "See https://yauaa.basjes.nl/README-Output.html",
+            "enum": ["Desktop", "Anonymized", "Unknown", "Mobile", "Tablet", "Phone", "Watch", "Virtual Reality", "eReader", "Set-top box", "TV", "Game Console", "Handheld Game Console", "Voice", "Robot", "Robot Mobile", "Spy", "Hacker"]
+        },
+        "deviceName": {
+            "description": "Example: Google Nexus 6",
+            "type": "string"
+        },
+        "deviceBrand": {
+            "description": "Example: Google",
+            "type": "string"
```

Comment:
I know that there's likely no specification on how long it can be. But in fact we always have an upper bound for strings (at least in Redshift) - it is just implcit 4096. Sometimes we can do better even with guesswork.

Reply:
I added `maxLength` for almost all the fields. Hopefully I took enough margin regarding the lengths of the fields, based on their names and some examples.